### ISSUE TITLE
gen_simple_client_config_xml incompatible with XML schema

### ISSUE THREAD

**Issue by dikim33** [oscar-cluster/oscar#154]
**Reported by bli on 26 Nov 2005 19:55 UTC**
```
The script gen_simple_client_config_xml currently
generates a config.xml that is incompatible with the
new config.xml schema that was recently checked into trunk.
```

**Comment by dikim33** [comment]
**Comment by efocht on 4 Aug 2006 08:45 UTC**
As Bernard has asked for comments... here is one:

In the very early days packages without config.xml were allowed. A default
config.xml was created on the fly by looking at the files in RPMS. I've never
seen a package without config.xml and I consider this bug irrelevant. The
corresponding code should be deleted as packages without config.xml are not
regarded as oscar packages (this is a statement inside wizard_prep). Close the
bug as "invalid", that seems appropriate.

Regards,
Erich

**Comment by dikim33** [comment]
**Comment by bli on 9 Aug 2006 02:48 UTC**
I agree with Erich's comments, so if I don't hear any complaints I will nuke this script from both trunk and branch-5-0.

**Comment by dikim33** [comment]
**Comment by valleegr on 19 Aug 2006 04:28 UTC**
I think the biggest problem is not to know if the config.xml is compliant to the schema/DTD but the fact that we do not even have such a schema/DTD!
First question: do we want an XML schema? or a DTD is good enough?

**Comment by dikim33** [comment]
**Comment by bli on 29 Aug 2006 04:13 UTC**
I'm closing this ticket, let's move discussions regarding XML schema/DTD to oscar-devel@lists.sourceforge.net.  Thanks!

**Comment by dikim33** [comment]
**Comment by bli on 29 Aug 2006 04:13 UTC**
(In [5321]) This commit is a merge from trunk -> branch-5-0:
   svn merge -r5319:5320 https://svn.oscar.openclustergroup.org/svn/oscar/trunk .

Closes #156 for milestone OSCAR 5.0.

Original commit message:
Fix for #156.  gen_simple_client_config_xml is no longer necessary since all OSCAR
packages should have a correct config.xml file included.
